packages/grid/src/vaadin-grid-sorter.js

```diff
@@ -170,12 +170,28 @@ class GridSorter extends ThemableMixin(DirMixin(PolymerElement)) {
   connectedCallback() {
     super.connectedCallback();
     this._isConnected = true;
+    this._grid = this._findHostGrid();
   }
 
   /** @protected */
   disconnectedCallback() {
     super.disconnectedCallback();
     this._isConnected = false;
+
+    if (!this.parentNode && this._grid && this.direction) {
```

Comment:
Is the `this.direction` check necessary? Could use a test case.

Reply:
Actually, it's not necessary. Removed the check.